dashboards-observability/public/components/event_analytics/utils/utils.tsx

```diff
@@ -318,3 +318,9 @@ export const lightenColor = (color: string, percent: number) => {
       .slice(1)
   );
 };
+
+export const filterDataConfigParameter = (parameter: Array<ConfigListEntry>) =>
+  parameter.filter((i: ConfigListEntry) => i.label);
+
+export const getRoundOf = (value: number, places: number) =>
+  (Math.round(value * 10 ** places) / 10 ** places).toFixed(places);
```

Comment:
I do not understand the purpose. Can't we use Math.ceil() if you need the upper limit?